angular-app/src/app/app.component.html

```diff
@@ -0,0 +1,3 @@
+<app-shell></app-shell>
+<app-home></app-home>
```

Comment:
Ideea app shell-ului este de a avea un container scrollable pentru toata aplicatia. Avem header-ul care este positionat fixed si un container scrollable unde incarcam restul applicatie. In cazul nostru, app-home si app-todo-add, trebuie sa fie  declarat in interiorul tag-ului app-shell.

Reply:
Ok. Am inteles logica structurii aplicatiei si am corectat. Multumesc pentru observatie.